Documentation/ApiOverview/ContentElements/Index.rst

```diff
@@ -183,12 +185,9 @@ To register such a plugin as content element you can use function
 ..  literalinclude:: _Plugins/_tt_content_plugin.php
     :caption: EXT:my_extension/Configuration/TCA/Overrides/tt_content.php
 
-By using `'list_type'` as second parameter is is also possible to add the plugin
-as a list type. See :ref:`plugins-list_type`.
-
-**Plugins** are a specific type of content elements. Plugins use the CType='list'.
-Each plugin has its own plugin type, which is used in the database field
-tt_content.list_type. The list_type could be understood as subtype of CType.
+..  deprecated:: 13.4
+    Setting the second parameter to any value but `CType`
```

Comment:
Only `list_type` is deprecated. You still can call that method without any deprecations for any TCA select type. Sure, it does not make sense, but it's possible. `addPlugin` looks a little bit like `addTcaSelectItem`, it just adds a new entry to a select type.